larpandoracontent/LArHelpers/LArInteractionTypeHelper.cc

```diff
@@ -437,6 +550,44 @@ std::string LArInteractionTypeHelper::ToString(const InteractionType interaction
     case BEAM_PARTICLE_KAON_PLUS: return "BEAM_PARTICLE_KAON_PLUS";
     case BEAM_PARTICLE_KAON_MINUS: return "BEAM_PARTICLE_KAON_MINUS";
     case BEAM_PARTICLE_OTHER: return "BEAM_PARTICLE_OTHER";
+    case BEAM_PARTICLE_PI_PLUS_PI_PLUS: return "BEAM_PARTICLE_PI_PLUS_PI_PLUS";
```

Comment:
Could ultimately consider going for one of our preprocessor list tricks here, with one table that is used to populate the enumeration and ToString function automatically.